src/exporter/ddprof_exporter.cc

```diff
@@ -28,8 +30,9 @@ extern "C" {
 static const int k_timeout_ms = 10000;
 static const int k_size_api_key = 32;
 
-static ddprof_ffi_CharSlice cpp_string_to_CharSlice(const std::string &str) {
-  return (ddprof_ffi_CharSlice){.ptr = (char *)str.c_str(), .len = str.size()};
+static ddprof_ffi_CharSlice
+cpp_string_to_CharSlice(const std::string_view &str) {
```

Comment:
Why not use function overloading instead of using a new function name ?

Reply:
Fixed